blivet/udev.py

```diff
@@ -143,8 +140,8 @@ def __is_blacklisted_blockdev(dev_name):
     if dev_name.startswith("ram") or dev_name.startswith("fd"):
         return True
 
-    if flags.installer_mode:
-        if any(re.search(expr, dev_name) for expr in INSTALLER_BLACKLIST):
+    if flags.blacklisted_devs:
+        if any(re.search(expr, dev_name) for expr in flags.blacklisted_devs):  # pylint: disable=not-an-iterable
```

Comment:
Yeah; so dlehman suggested I just make udev.device_name_blacklist and set it to be an empty list by default; enable_installer_mode() can set it to the list of regexes. I think I prefer that , so that's what I've implemented.